apps/meteor/ee/app/livechat-enterprise/server/lib/Helper.js

```diff
@@ -249,7 +250,7 @@ export const updatePriorityInquiries = (priority) => {
 
 	const { _id: priorityId } = priority;
 	LivechatRooms.findOpenByPriorityId(priorityId).forEach((room) => {
-		updateInquiryQueuePriority(room._id, priority);
+		Promise.await(updateInquiryQueuePriority(room._id, priority));
```

Comment:
If syncronization isn't required, then maybe we can benifit from Promise.all() here